lib/licensee/projects/uri_project.rb

```diff
@@ -0,0 +1,25 @@
+# URI-based project
+#
+# Analyzes a given URI for license information
+require 'net/http'
+require 'uri'
+
+module Licensee
+  class UriProject < Project
+    def initialize(path, **args)
+      @uri = URI(path)
+      raise UnsupportedProject if @uri.scheme !~ /^https?$/
+      super(**args)
+    end
+
+    def load_file(file)
```

Comment:
For every other project-type, `load_file` take a hash with key `name`, which is relative to the project root. Perhaps to mirror that behavior, `load_file` should expect the same hash, but URI.join the relative path to the root URI?